tinywallet/tinywallet/app.py

```diff
@@ -513,11 +426,27 @@ def getButton(self, size, text, tracked=True):
             self.trackedCssItems.append(button)
         return button
 
-    def home_(self):
-        """
-        Go to the home screen.
+    def home_(self, screen=None):
         """
+        Go to the home screen or set a new home screen.
+
+        Args:
+            screen (Screen | tuple(Screen)): Behavior will differ depending on
+                type passed. If no screen is passed, the last home screen will
+                be used. If screen is a Screen, it will be set as the new home
+                screen. If screen is a tuple of Screen, The first Screen in the
+                tuple will be set to the home screen, and the rest will be
+                stacked.
+        """
+        stacks = tuple()
+        if isinstance(screen, tuple):
```

Comment:
Any reason to specifically require a `tuple` here? Any `Sequence` should do:
```
        if isinstance(screen, collections.abc.Sequence):
```

Reply:
What's wrong with specifying a tuple?

Reply:
It seems unnecessarily restrictive: you might have a list of screens or some other sequence, and would have to convert that to a tuple. But if that's not gonna happen, no worries.

Reply:
I guess I could see how it might be nice to allow other sequence types. I might follow up.